src/Sarif/Schemata/sarif-schema.json

```diff
@@ -487,6 +487,118 @@
             }
         },
 
+        "externalProperties": {
+            "type": "object",
+            "properties": {
+
+                "$schema": {
+                    "description": "tbd",
+                    "type": "string"
+                },
+
+                "version": {},
+
+                "instanceGuid": {},
```

Comment:
>instanceGuid [](start = 17, length = 12)

this s/be called 'guid'

Reply:
@lgolding, we need a call asap on naming for 'guid' members. Will we consistently prefer 'instanceGuid' or 'guid'? What say you, Dobbin?

---
In reply to: [263972946](https://github.com/Microsoft/sarif-sdk/pull/1342#discussion_r263972946) [](ancestors = 263972946)